frappe/data_migration/doctype/data_migration_plan/data_migration_plan.py

```diff
@@ -8,6 +7,20 @@
 from frappe.custom.doctype.custom_field.custom_field import create_custom_field
 from frappe.model.document import Document
 
+
+def get_mapping_module(module, mapping_name):
+	app_name = frappe.db.get_value("Module Def", module, "app_name")
+	mapping_name = frappe.scrub(mapping_name)
+	module = frappe.scrub(module)
+
+	try:
+		return frappe.get_module(
+			f"{app_name}.{module}.data_migration_mapping.{mapping_name}"
+		)
+	except ImportError:
+		return None
```

Comment:
Does this only make sense within the Migration Plan context? Cause then it could just be a staticmethod.